code/game/objects/items/storage/marine_boxes.dm

```diff
@@ -1,47 +1,78 @@
+	icon = 'icons/obj/items/storage/storage.dmi'
+	icon_state = "case"
+	foldable = /obj/item/stack/sheet/wood
+
+// Copied from C:\Dev\SS13\TerraGov-Marine-Corps\code\game\objects\items\storage\storage.dm
+/obj/item/storage/box/crate/attack_self(mob/user)
+	//Clicking on itself will empty it, if it has the verb to do that.
+	if(allow_quick_empty)
+		quick_empty()
+		return
+
+	//Otherwise we'll try to fold it.
+	if ( length(contents) )
+		return
+
+	if ( !ispath(foldable) )
+		return
+
+	// Close any open UI windows first
+	for(var/mob/M in content_watchers)
+		close(M)
+
+	// Now make the plank
+	to_chat(user, span_notice("You broken [src] into a plank."))
+	new foldable(get_turf(src))
+	qdel(src)
+
+/obj/item/storage/box/crate/update_icon_state()
+	icon_state = initial(icon_state)
+	if(!length(contents))
+		icon_state = "empty_case"
+/obj/item/storage/box/crate/heavy_armor
 	name = "\improper B-Series defensive armor crate"
 	desc = "A large case containing an experiemental suit of B18 armor for the discerning specialist."
-	icon = 'icons/Marine/marine-weapons.dmi'
 	icon_state = "armor_case"
 	w_class = WEIGHT_CLASS_HUGE
 	storage_slots = 3
 	can_hold = list() //Nada. Once you take the stuff out it doesn't fit back in.
-	foldable = null
+	//foldable = null
 
-/obj/item/storage/box/heavy_armor/Initialize(mapload, ...)
+/obj/item/storage/box/crate/heavy_armor/Initialize(mapload, ...)
 	. = ..()
 	new /obj/item/clothing/gloves/marine/specialist(src)
 	new /obj/item/clothing/suit/storage/marine/specialist(src)
 	new /obj/item/clothing/head/helmet/marine/specialist(src)
 
-/obj/item/storage/box/grenade_system
+/obj/item/storage/box/crate/grenade_system
 	name = "\improper M92 grenade launcher case"
 	desc = "A large case containing a heavy-duty multi-shot grenade launcher, the Armat Systems M92. Drag this sprite into you to open it up!\nNOTE: You cannot put items back inside this case."
-	icon = 'icons/Marine/marine-weapons.dmi'
 	icon_state = "grenade_case"
 	w_class = WEIGHT_CLASS_HUGE
 	storage_slots = 2
 	slowdown = 1
 	can_hold = list() //Nada. Once you take the stuff out it doesn't fit back in.
-	foldable = null
+	//foldable = null
 
-/obj/item/storage/box/grenade_system/Initialize(mapload, ...)
+/obj/item/storage/box/crate/grenade_system/Initialize(mapload, ...)
 	. = ..()
 	new /obj/item/weapon/gun/grenade_launcher/multinade_launcher(src)
 	new /obj/item/storage/belt/grenade/b17(src)
 
-/obj/item/storage/box/rocket_system
+/obj/item/storage/box/crate/rocket_system
 	name = "\improper M5 RPG crate"
 	desc = "A large case containing a heavy-caliber antitank missile launcher and missiles. Drag this sprite into you to open it up!\nNOTE: You cannot put items back inside this case."
-	icon = 'icons/Marine/marine-weapons.dmi'
 	icon_state = "rocket_case"
 	w_class = WEIGHT_CLASS_HUGE
 	storage_slots = 6
 	slowdown = 1
 	can_hold = list() //Nada. Once you take the stuff out it doesn't fit back in.
-	foldable = null
+	//foldable = null
```

Comment:
? bruh why you doing this, I'm not going to comment on every single one.

Reply:
I removed them to ensure they produce a plank of wood when folded. I'll get rid of the comment